### PR TITLE
Added Spectrum chat cask

### PR DESCRIPTION
**Pull request by briannhinton**
New cask. No stable version exists, and I'm following the suggestion in the guide regarding that by submitting it here.

Cask's name is included. But no version is specified. 

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by mrbrianhinton** [comment]
Corrected the errors. Styling docs were a bit unclear on how to comment. Took a few tries to understand it.

File: Casks/spectrum.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'spectrum' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/spectrum.rb]: This needs a version (`1.0.1`).

File: Casks/spectrum.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/spectrum.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

File: Casks/spectrum.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+  # github.com/withspectrum was verified as official when first introduced to the cask
+  url 'https://github.com/withspectrum/spectrum/releases/download/desktop-v1.0.1/Spectrum-1.0.1-mac.zip'
```
Comment (vitorgalvao) [inline Casks/spectrum.rb]: Add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

File: Casks/spectrum.rb (hunk 1 above)
Comment (vitorgalvao) [reply Casks/spectrum.rb]: > parameters specified in comments

What comments?

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**